casper/src/main/scala/coop/rchain/casper/blocks/proposer/Proposer.scala

```diff
@@ -164,9 +165,7 @@ object Proposer {
         // TODO: temp this is done after validation
 //        MultiParentCasper.handleValidBlock(b) >>
         // broadcast hash to peers
-        CommUtil[F].sendBlockHash(b.blockHash, b.sender) >>
-        // Publish event
-        EventPublisher[F].publish(MultiParentCasper.createdEvent(b))
```

Comment:
`MultiParentCasper.createdEvent` was used only here. Maybe remove it?

Reply:
Good catch! I'll remove it, together with `RChainEvent` definition (https://github.com/rchain/rchain/pull/3758/commits/c6b17b305dd58f48c4cd83529a76a6bd9af7c22d).